_router/routes.js.md

```diff
@@ -22,6 +22,6 @@ module.exports = function(addRoute) {
 
 The callback function takes 3 parameters:
 
-- The URL for the route, this can also be a regular expression to match
-- The controller [name](/controller#name) and the [action](/controller#action), 'controllerName#action'
+- The URL for the route, which can also be a regular expression
+- The controller [name](/controller#name) and [action](/controller#action), 'controllerName#action'
 - *optional* An object of options to pass to the router.  If the object contains a redirect, no controller information is required.
```

Comment:
@saponifi3d Can you add an example of an object with redirect that omits controller information?

e.g.
`addRoute('some/redirect', { redirect: '/foo' });`

?